e2e/app/CreateAccountPage.ts

```diff
@@ -0,0 +1,218 @@
+      }
+    }
+    return newUserName;
+  }
+
+  // select Institution Affiliation from a dropdown
+  public async selectInstitution(selectTextValue: string) {
+    const dropdown = new SelectComponent(this.puppeteerPage);
+    await dropdown.select(selectTextValue);
+  }
+
+  public async getInstitutionValue() {
+    const dropdown = new SelectComponent(this.puppeteerPage);
+    return await dropdown.getSelectedValue();
+  }
+
+  // select Education Level from a dropdown
+  public async selectEducationLevel(selectTextValue: string) {
+    const dropdown = new SelectComponent(this.puppeteerPage, FIELD_LABEL.EDUCATION_LEVEL);
+    await dropdown.select(selectTextValue);
+  }
+
+  // select Year of Birth from a dropdown
+  public async selectYearOfBirth(year: string) {
+    const dropdown = new SelectComponent(this.puppeteerPage, FIELD_LABEL.YEAR_OF_BIRTH);
+    await dropdown.select(year);
+  }
+
+  // Combined steps to make test code cleaner and shorter
+
+  // Step 1: Enter Invitation key
+  public async fillOutInvitationKey(invitationKey: string) {
+    await this.getInvitationKeyInput()
+    .then(invitationKeyInput => invitationKeyInput.type(invitationKey))
+    .then(() => this.getNextButton())
+    .then(submitButton => submitButton.click());
+  }
+
+  // Step 2: Accepting Terms of Use and Privacy statement.
+  public async acceptTermsOfUseAgreement() {
+    await this.getPrivacyStatementCheckbox();
+    await this.getTermsOfUseCheckbox();
+    await this.getNextButton();
+
+    await this.scrollToLastPdfPage();
+
+    // check by click on label works
+    await (await this.getPrivacyStatementLabel()).click();
+    await (await this.getTermsOfUseLabel()).click();
+  }
+
+  // Step 3: Enter user default information
+  public async fillOutUserInformation() {
+    const newUserName = await this.fillInFormFields(defaultFieldValues);
+    await (await this.getResearchBackgroundTextarea()).type(faker.lorem.word());
+    // a different dropdown selection depending on Yes or No radiobutton was selected
+    await (await this.areYouAffiliatedRadioButton(true)).click();
+    await (await this.getInstitutionNameInput()).type(faker.company.companyName());
+    await this.selectInstitution(INSTITUTION_AFFILIATION.EARLY_CAREER_TENURE_TRACK_RESEARCHER);
+    await this.puppeteerPage.waitFor(1000);
+    return newUserName;
+  }
+
+  // Step 4: Enter demographic survey default information (All Survey Fields are optional)
+  public async fillOutDemographicSurvey() {
+    // Find and check on all checkboxes with same label: Prefer not to answer
+    const targetXpath = '//*[normalize-space(text())="Prefer not to answer"]/ancestor::node()[1]/input[@type="checkbox"]';
+    await this.puppeteerPage.waitForXPath(targetXpath, { visible: true });
+    const checkboxs = await this.puppeteerPage.$x(targetXpath);
+    for (const ck of checkboxs) {
+      await ck.click();
+    }
+    // Select year of birth 1955
```

Comment:
nit: I would remove this kind of comment (here and below). This is just restating what the next line of code is doing, which I could have learned just as easily by reading the code. I would use comments for higher level information - or alternatively to explain "why" we are doing something. The comment 10 lines above is fine because the code is pretty hairy and difficult to read (arguably, you could alternatively eliminate the comment and breakup the xpath selection to be easier to read).

Reply:
removed.